<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
	<HEAD>
	<META http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
	<TITLE>no title [MT]</TITLE>
	<link rel="stylesheet" href="http://www.mutationtaster.org/css.css" type="text/css">
	<style type="text/css">
	<!--
		.mm {	color: black }
		.m {	background-color: black;	color: white }
		.c {	background-color: #CCCCCC; 	color: white }
		a:visited {color: #0033FF;}
		.error {color: red;}
		.box { border: 1px solid #999999 }
	-->
	</style>
	</HEAD>
	<BODY>

	<table  boundary="0" width="100%" cellspacing="20">
		<tr>
			<td style="text-align:center">
				<img src="http://doro.charite.de/MutationTaster/MutationTaster_small.png" alt="Yum, tasty mutations..." align="left">
			</td>
			<td><h1>mutation t<span class="tasty">@</span>sting</h1></td>
			<td><A HREF="http://doro.charite.de/MutationTaster/info/documentation.html" TARGET="_blank">documentation</A></td>
		</tr>
	</table> 
	<!--pred 9|0.999999879175341|without_aae-->
	<table border="0">
			<tr>
			<td><h3 class="green">Prediction</h3></td>
			<td width="400">
			<h3 class="green">polymorphism</h3></td>
			<td  style="vertical-align: middle">
			<b class="green">Model: <i>without_aae</i>, prob: 0.999999879175341  (classification due to TGP/ExAC,
			<a href="http://www.mutationtaster.org/info/documentation.html#specialcases" TARGET="new">real probability</A> is shown anyway)</b>
			&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<small>(<a href="http://doro.charite.de/MutationTaster/info/documentation.html#bayes" target="_new">explain</a>)</small></td>
			</TR>
			<tr><td><strong>Summary</strong> </td>
     <td><ul> <li><strong>homozygous in TGP or ExAC</strong></li>  <li><strong>protein features (might be) affected</strong></li>  <li><strong>splice site changes</strong></li> </ul></td>
<td><A HREF="http://www.mutationtaster.org/cgi-bin/MutationTaster/MutationTaster69.cgi?transcript_stable_id_text=ENST00000379370&sequence_type=gDNA&start_insdel=28669&end_insdel=28672">hyperlink</A></td></tr>	<tr>
			<th>analysed issue</th>
			<th colspan="2" >analysis result</th>
		</tr> <tr  bgcolor="#CCCCCC"><td  >name of alteration</td> <td colspan="2" >no title</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >alteration (phys. location)</td> <td colspan="2" >chr1:984172_984173delAG</td></tr>
 <tr  bgcolor="#CCCCCC"><td  >HGNC symbol</td> <td colspan="2" ><A TARGET="_blank"
		HREF="http://www.genedistiller.org/cgi-bin/GeneDistiller_results.cgi?show_interaction=1&show_hpo=1&show_pathways=1&txid=9606&order=start_pos&show_generifs=1&show_mgd_phenotypes=1&show_GO=1&show_pfam=1&show_transcripts=1&show_omim=1&show_synonyms=1&show_interpro=1&show_paralogs=1&show_proteinfamilies=1&genesymbol=AGRN">AGRN</A></td></tr>
 <tr  bgcolor="#FFFFFF"><td  >Ensembl transcript ID</td> <td colspan="2" ><A HREF="http://grch37.ensembl.org//Homo_sapiens/transview?db=core;transcript=ENST00000379370" TARGET="_blank">ENST00000379370</A></td></tr>
 <tr  bgcolor="#CCCCCC"><td  >Genbank transcript ID</td> <td colspan="2" ><A HREF="http://www.ncbi.nlm.nih.gov/nuccore/NM_198576" TARGET="_blank">NM_198576</A></td></tr>
 <tr  bgcolor="#FFFFFF"><td  >UniProt peptide</td> <td colspan="2" ><A TARGET="_blank" HREF="http://www.uniprot.org/uniprot/O00468">O00468</A></td></tr>
 <tr  bgcolor="#CCCCCC"><td  >alteration type</td> <td colspan="2" >deletion</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >alteration region</td> <td colspan="2" >intron</td></tr>
 <tr  bgcolor="#CCCCCC"><td  >DNA changes</td> <td colspan="2" >g.28670_28671delAG<br></td></tr>
 <tr  bgcolor="#FFFFFF"><td class="grey" >AA changes</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#CCCCCC"><td class="grey" >position(s) of altered AA<br><small>if AA alteration in CDS</small></td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#FFFFFF"><td class="grey" >frameshift</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#CCCCCC"><td  >known variant</td> <td colspan="2" >Reference ID: <A HREF="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=140904842">rs140904842</A>

<table border="0" cellpadding="4" cellspacing="0">
				<tr><td class="box">database</td><td class="box" style="background-color: #76EEC6">homozygous (-/-)</td><td class="box">heterozygous</td><td class="box">allele carriers</td></tr>

<tr><td class="box">1000G</td><td class="box" style="background-color: #76EEC6"><b>2135</b></td><td class="box">340</td><td class="box">2475</td></TR>

<tr><td class="box"><A HREF="http://exac.broadinstitute.org/variant/1-984171-CAG-C">ExAC</A></td><td class="box" style="background-color: #76EEC6">-</td><td class="box">-</td><td class="box">-</td></tr></table>
</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >regulatory features</td> <td colspan="2" >H3K36me3, Histone, Histone 3 Lysine 36 Tri-Methylation </td></tr>
 <tr  bgcolor="#CCCCCC"><td  >phyloP / phastCons</td> <td colspan="2" ><table border="0" cellpadding="4"><tr><td></td><td>PhyloP</td><td>PhastCons</td></tr>
<tr><td>(flanking)</td><td>0.2</td><td>0.006</td></tr><tr><td></td><td>-1.159</td><td>0.003</td></tr><tr><td></td><td>0.178</td><td>0.003</td></tr><tr><td>(flanking)</td><td>0.309</td><td>0.002</td></tr></table> <A HREF="http://www.mutationtaster.org/info/documentation.html#phylop" target="_new"><small>explain score(s)</small></A> and/or inspect your position(s) in <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?org=Human&db=hg19&position=chr1:984171-984174" target="new">in UCSC Genome Browser</a></td></tr>
 <tr  bgcolor="#FFFFFF"><td  >splice sites</td> <td colspan="2" > <table><tr><td>effect</td><td>gDNA position</td><td>score</td><td>wt detection sequence</td><td>&nbsp;exon-intron border</td></tr> <tr><td>Donor marginally increased</td><td>28662</td><td>wt: 0.9900 / mu: 0.9932 (marginal change - not scored)</td><td class='code'>wt: AAAAAAAAAAAGCAG<br>mu: AAAAAAAAAAAGCGG</td><td class='code'>&nbsp;AAAA|aaaa</td></tr><tr><td>Donor marginally increased</td><td>28661</td><td>wt: 0.6598 / mu: 0.6855 (marginal change - not scored)</td><td class='code'>wt: AAAAAAAAAAAAGCA<br>mu: AAAAAAAAAAAAGCG</td><td class='code'>&nbsp;AAAA|aaaa</td></tr><tr><td>Donor gained</td><td>28671</td><td>0.36</td><td class='code'>mu: AAAAGCGGGGCGGTT</td><td class='code'>&nbsp;GCGG|ggcg</td></tr><tr><td>Donor gained</td><td>28664</td><td>0.62</td><td class='code'>mu: AAAAAAAAAGCGGGG</td><td class='code'>&nbsp;AAAA|aaag</td></tr></table></td></tr>
 <tr  bgcolor="#CCCCCC"><td  >distance from splice site</td> <td colspan="2" >74</td></tr>
 <tr  bgcolor="#FFFFFF"><td class="grey" >Kozak consensus sequence altered?</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#CCCCCC"><td class="grey" >conservation<br><small>protein level for non-synonymous changes</small></td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >protein features</td> <td colspan="2" ><table><tr><td>start (aa)</td><td>end (aa)</td><td>feature</td><td>details</td><td>&nbsp;</td></tr><tr><td>1372</td><td>1548</td><td>DOMAIN</td><td>Laminin G-like 1.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1519</td><td>1519</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1548</td><td>1548</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1549</td><td>1586</td><td>DOMAIN</td><td>EGF-like 2.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1553</td><td>1553</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1558</td><td>1558</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1564</td><td>1564</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1574</td><td>1574</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1576</td><td>1576</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1585</td><td>1585</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1588</td><td>1625</td><td>DOMAIN</td><td>EGF-like 3.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1592</td><td>1592</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1597</td><td>1597</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1603</td><td>1603</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1613</td><td>1613</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1615</td><td>1615</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1624</td><td>1624</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1635</td><td>1818</td><td>DOMAIN</td><td>Laminin G-like 2.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1814</td><td>1853</td><td>DOMAIN</td><td>EGF-like 4.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1818</td><td>1818</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1826</td><td>1826</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1832</td><td>1832</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1841</td><td>1841</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1843</td><td>1843</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1852</td><td>1852</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>1864</td><td>2042</td><td>DOMAIN</td><td>Laminin G-like 3.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>2016</td><td>2016</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr><tr><td>2042</td><td>2042</td><td>DISULFID</td><td>By similarity.</td><td>might get lost (downstream of altered splice site)</td></tr></table></td></tr>
 <tr  bgcolor="#CCCCCC"><td class="grey" >length of protein</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#FFFFFF"><td class="grey" >AA sequence altered</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#CCCCCC"><td class="grey" >position of stopcodon in wt / mu CDS</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#FFFFFF"><td class="grey" >position (AA) of stopcodon in wt / mu AA sequence</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#CCCCCC"><td class="grey" >position of stopcodon in wt / mu cDNA</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#FFFFFF"><td class="grey" >poly(A) signal</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#CCCCCC"><td class="grey" >conservation<br><small>nucleotide level for all changes - no scoring up to now</small></td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >position of start ATG in wt / mu cDNA</td> <td colspan="2" >51 / 51</td></tr>
 <tr  bgcolor="#CCCCCC"><td  >chromosome</td> <td colspan="2" >1</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >strand</td> <td colspan="2" >1</td></tr>
 <tr  bgcolor="#CCCCCC"><td  >last intron/exon boundary</td> <td colspan="2" >6031</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >theoretical NMD boundary in CDS</td> <td colspan="2" >5930</td></tr>
 <tr  bgcolor="#CCCCCC"><td  >length of CDS</td> <td colspan="2" >6138</td></tr>
 <tr  bgcolor="#FFFFFF"><td class="grey" >coding sequence (CDS) position</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#CCCCCC"><td class="grey" >cDNA position<br><small>(for ins/del: last normal base / first normal base)</small></td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >gDNA position<br><small>(for ins/del: last normal base / first normal base)</small></td> <td colspan="2" >28669 / 28672</td></tr>
 <tr  bgcolor="#CCCCCC"><td  >chromosomal position<br><small>(for ins/del: last normal base / first normal base)</small></td> <td colspan="2" >984171 / 984174</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >original gDNA sequence snippet</td> <td colspan="2" class="code">AAAAAAAAAAAAAAAAAAGC<b class="blue">AG</b>GGGGCGGTTTCAGGGATAAA</td></tr>
 <tr  bgcolor="#CCCCCC"><td  >altered gDNA sequence snippet</td> <td colspan="2" class="code">AAAAAAAAAAAAAAAAAAGC<b class="blue"></b>GGGGCGGTTTCAGGGATAAA</td></tr>
 <tr  bgcolor="#FFFFFF"><td class="grey" >original cDNA sequence snippet</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#CCCCCC"><td class="grey" >altered cDNA sequence snippet</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >wildtype AA sequence</td> <td colspan="2" class="code"><span class="code">MAGRSHPGPL RPLLPLLVVA ACVLPGAGGT CPERALERRE EEANVVLTGT VEEILNVDPV <br>QHTYSCKVRV WRYLKGKDLV ARESLLDGGN KVVISGFGDP LICDNQVSTG DTRIFFVNPA <br>PPYLWPAHKN ELMLNSSLMR ITLRNLEEVE FCVEDKPGTH FTPVPPTPPD ACRGMLCGFG <br>AVCEPNAEGP GRASCVCKKS PCPSVVAPVC GSDASTYSNE CELQRAQCSQ QRRIRLLSRG <br>PCGSRDPCSN VTCSFGSTCA RSADGLTASC LCPATCRGAP EGTVCGSDGA DYPGECQLLR <br>RACARQENVF KKFDGPCDPC QGALPDPSRS CRVNPRTRRP EMLLRPESCP ARQAPVCGDD <br>GVTYENDCVM GRSGAARGLL LQKVRSGQCQ GRDQCPEPCR FNAVCLSRRG RPRCSCDRVT <br>CDGAYRPVCA QDGRTYDSDC WRQQAECRQQ RAIPSKHQGP CDQAPSPCLG VQCAFGATCA <br>VKNGQAACEC LQACSSLYDP VCGSDGVTYG SACELEATAC TLGREIQVAR KGPCDRCGQC <br>RFGALCEAET GRCVCPSECV ALAQPVCGSD GHTYPSECML HVHACTHQIS LHVASAGPCE <br>TCGDAVCAFG AVCSAGQCVC PRCEHPPPGP VCGSDGVTYG SACELREAAC LQQTQIEEAR <br>AGPCEQAECG SGGSGSGEDG DCEQELCRQR GGIWDEDSED GPCVCDFSCQ SVPGSPVCGS <br>DGVTYSTECE LKKARCESQR GLYVAAQGAC RGPTFAPLPP VAPLHCAQTP YGCCQDNITA <br>ARGVGLAGCP SACQCNPHGS YGGTCDPATG QCSCRPGVGG LRCDRCEPGF WNFRGIVTDG <br>RSGCTPCSCD PQGAVRDDCE QMTGLCSCKP GVAGPKCGQC PDGRALGPAG CEADASAPAT <br>CAEMRCEFGA RCVEESGSAH CVCPMLTCPE ANATKVCGSD GVTYGNECQL KTIACRQGLQ <br>ISIQSLGPCQ EAVAPSTHPT SASVTVTTPG LLLSQALPAP PGALPLAPSS TAHSQTTPPP <br>SSRPRTTASV PRTTVWPVLT VPPTAPSPAP SLVASAFGES GSTDGSSDEE LSGDQEASGG <br>GSGGLEPLEG SSVATPGPPV ERASCYNSAL GCCSDGKTPS LDAEGSNCPA TKVFQGVLEL <br>EGVEGQELFY TPEMADPKSE LFGETARSIE STLDDLFRNS DVKKDFRSVR LRDLGPGKSV <br>RAIVDVHFDP TTAFRAPDVA RALLRQIQVS RRRSLGVRRP LQEHVRFMDF DWFPAFITGA <br>TSGAIAAGAT ARATTASRLP SSAVTPRAPH PSHTSQPVAK TTAAPTTRRP PTTAPSRVPG <br>RRPPAPQQPP KPCDSQPCFH GGTCQDWALG GGFTCSCPAG RGGAVCEKVL GAPVPAFEGR <br>SFLAFPTLRA YHTLRLALEF RALEPQGLLL YNGNARGKDF LALALLDGRV QLRFDTGSGP <br>AVLTSAVPVE PGQWHRLELS RHWRRGTLSV DGETPVLGES PSGTDGLNLD TDLFVGGVPE <br>DQAAVALERT FVGAGLRGCI RLLDVNNQRL ELGIGPGAAT RGSGVGECGD HPCLPNPCHG <br>GAPCQNLEAG RFHCQCPPGR VGPTCADEKS PCQPNPCHGA APCRVLPEGG AQCECPLGRE <br>GTFCQTASGQ DGSGPFLADF NGFSHLELRG LHTFARDLGE KMALEVVFLA RGPSGLLLYN <br>GQKTDGKGDF VSLALRDRRL EFRYDLGKGA AVIRSREPVT LGAWTRVSLE RNGRKGALRV <br>GDGPRVLGES PVPHTVLNLK EPLYVGGAPD FSKLARAAAV SSGFDGAIQL VSLGGRQLLT <br>PEHVLRQVDV TSFAGHPCTR ASGHPCLNGA SCVPREAAYV CLCPGGFSGP HCEKGLVEKS <br>AGDVDTLAFD GRTFVEYLNA VTESEKALQS NHFELSLRTE ATQGLVLWSG KATERADYVA <br>LAIVDGHLQL SYNLGSQPVV LRSTVPVNTN RWLRVVAHRE QREGSLQVGN EAPVTGSSPL <br>GATQLDTDGA LWLGGLPELP VGPALPKAYG TGFVGCLRDV VVGRHPLHLL EDAVTKPELR <br>PCPTP*</span></td></tr>
 <tr  bgcolor="#CCCCCC"><td class="grey" >mutated AA sequence</td> <td colspan="2" class="grey">N/A</td></tr>
 <tr  bgcolor="#FFFFFF"><td  >speed</td> <td colspan="2" >0.07 s</td></tr>
	<tr  bgcolor="#CCCCCC"><td></td><td colspan="2" >
	<FORM name="feedback_form" action="/cgi-bin/feedback.cgi" target="feedback" method="POST" onClick="document.getElementById('feedback').style.height='250px';">
	<INPUT TYPE="submit" value="Report bugs / help us to improve MutationTaster!" style="background-color: blue; color: white">
	<input type="hidden" name="transcript" value="379370">
	<input type="hidden" name="gdna_start" value="28669">
	<input type="hidden" name="gdna_end" value="28672">
	<input type="hidden" name="new_bases" value="">
	<input type="hidden" name="prediction" value="polymorphism">
	</FORM></td></tr></table>
	
	<IFRAME name="feedback" id="feedback" width="100%" height="80" frameborder="no"></IFRAME>
	All positions are in basepairs (bp) if not explicitly stated differently.<br>
	AA/aa: amino acid; CDS: coding sequence; mu: mutated; NMD: nonsense-mediated mRNA decay; nt: nucleotide; wt: wildtype; TGP: 1000 Genomes Project<br>
	</BODY></HTML>
